pages/applications/create.tsx

```diff
@@ -0,0 +1,197 @@
+import { Form, DatePicker, Select, Button } from 'antd';
+import Title from 'antd/lib/typography/Title';
+import { useRouter } from 'next/router';
+import { useState } from 'react';
+import useSWR from 'swr';
+import applicationsApi from '../../api/applicationsApi';
+import companiesApi, { COMPANIES_API_ENDPOINT } from '../../api/companiesApi';
+import rolesApi, { ROLES_API_ENDPOINT } from '../../api/rolesApi';
+import CompanyOption from '../../components/CompanyOption';
+import CreateCompanyForm from '../../components/forms/CreateCompanyForm';
+import CreateRoleForm from '../../components/forms/CreateRoleForm';
+import { ApiResponse } from '../../types/apiResponse';
+import { CompanyAutocompleteOption, CompanyListData, CompanyQueryParams } from '../../types/company';
+import { RoleAutocompleteOption, RoleData, RoleListData, RoleQueryParams } from '../../types/role';
+import { Nullable } from '../../types/utils';
+import { createJsonResponse } from '../../utils/http/httpHelpers';
+import { splitByWhitespaces } from '../../utils/strings/formatters';
+import moment from 'moment';
+import { roleTypeToDisplayStringMap } from '../../utils/role/roleUtils';
+
+function ApplicationCreate() {
+  const router = useRouter();
+
+  const [isCreateCompanyFormOpen, setIsCreateCompanyFormOpen] = useState<boolean>(false);
+  const [isCreateRoleFormOpen, setIsCreateRoleFormOpen] = useState<boolean>(false);
+
+  const [companySearchParams, setCompanySearchParams] = useState<CompanyQueryParams>({ companyNames: [] });
+  const [selectedCompany, setSelectedCompany] = useState<Nullable<CompanyListData>>(null);
+
+  const [roleSearchParams, setRoleSearchParams] = useState<RoleQueryParams>({ searchWords: [] });
+  const [selectedRole, setSelectedRole] = useState<Nullable<RoleData>>(null);
+  const [applicationDate, setApplicationDate] = useState<Nullable<moment.Moment>>(moment(new Date()));
+
+  const [shouldShowValidationErrors, setShouldShowValidationErrors] = useState(false);
+
+  const { data: companiesData, mutate: mutateCompaniesData } = useSWR<ApiResponse<CompanyListData[]>>(
+    [COMPANIES_API_ENDPOINT, companySearchParams],
+    (url: string, companySearchParams: CompanyQueryParams) => companiesApi.getCompanies(companySearchParams),
+  );
+  const companies = companiesData?.payload || [];
+  const companyOptions: CompanyAutocompleteOption[] = companies.map((company) => ({
+    company,
+    label: <CompanyOption company={company} />,
+    value: company.name,
+  }));
+  const companyOptionsWithAdd: CompanyAutocompleteOption[] = [
+    ...companyOptions,
+    { company: null, label: <div>Add new company</div>, value: 'Add new company' },
```

Comment:
Can we add a `+` icon and make this a bit gray?

Reply:
![image](https://user-images.githubusercontent.com/75519136/190869637-05fc6de7-bf62-48f1-ba6a-e8314a59e841.png)

This is what I can do without trying to manipulate the inner antd CSS, feel free to remove it if you want.